python/tests/rest_api/rest_api_output_openapi_to_file..feature

```diff
@@ -0,0 +1,15 @@
+# WARNING - DO NOT EDIT - YOUR CHANGES WILL NOT BE PROTECTED.
+# This file is auto-generated by the aac gen-plugin and may be overwritten.
+
+
+Feature: Output OpenAPI to file.
+
+  Scenario: Output the OpenAPI specification to file.
+  
+      Given The {{gen-openapi-spec.input.output_directory}} contains a valid directory.
+      When The AaC app is run with the generate-openapi-spec command.
+      Then The AaC app is run with the generate-openapi-spec command.
```

Comment:
Also, not something to correct here, but this is pulling the `when` and populating in both the `when` and `then` entries. It's in my list of corrections for the feature file template work with `gen-gherkin`